src/views/includes/header.slim

```diff
@@ -0,0 +1,10 @@
+/ Header
```

Comment:
This can sometimes mess up the compilation.  I don't think we need a comment that's the same as the file name.